yaml_files/simphony_metadata.yml

```diff
@@ -418,5 +418,36 @@ CUDS_KEYS:
     definition: Multiphase model
 
   ELECTROSTATIC_MODEL:
-    parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CFD, CUBA.MOLECULAR_DYNAMICS, CUBA.DEM]
+
+  BASIS:
+    definition: Space basis vectors
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.VECTOR:
+      shape: (3, 3)
```

Comment:
@kitchoi I think we have to add the `shape` defined in this file to the `shape` defined in the `cuba.yaml` file and then validate the value. Therefore, a shape of `(3)` for `BASIS.CUBA.VECTOR` would imply a *real* shape of `(3, 3)` for it.